Resources/Maps/_Goobstation/cluster.yml

```
@@ -57703,6 +57703,13 @@ entities:
     - type: Transform
       pos: 0.4215492,21.564632
       parent: 2
+- proto: MaterialSilo
+  entities:
+  - uid: 12949
```

yamllint (1.35.1)

[error] wrong indentation: expected 4 but found 2

(indentation)